api/v1beta1/verticadb_types.go

```diff
@@ -39,10 +39,12 @@ const VerticaDBAPIVersion = "vertica.com/v1beta1"
 type VerticaDBSpec struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:=IfNotPresent
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// This dictates the image pull policy to use
```

Comment:
Some of the data structures that this one refers to don't have the new markers.  Why is that?  I'm thinking ones like LocalStorage, CommunalStorage, etc.

Reply:
I put this new marker above every field of VerticaDBSpec so that information about the field will be used to fill the specDescriptors field in the CSV as that part was missing leading to scorecard tests failure

Reply:
But why don't we include the fields in LocalStorage and CommunalStorage?  Can you fill those in when you create a CR from the webUI?

Reply:
Yeah you are right we can also add that to fields in data structures. It is just that to pass tests  adding that marker to just fields inside VerticaDBSpec is enough. And yes we can already fill them all but local.requestSize from the webUI.

Reply:
I think I will add the marker to all subfields for spec and status.